### PR TITLE
Add documentation for generator questions and set up.

### PR DESCRIPTION
**Pull request by jbalboni**
https://github.com/department-of-veterans-affairs/vets-contrib/issues/68

### CODE REVIEW

File: docs/GeneratorOptions.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+This is the url your application will live at. In your React apps you will likely have multiple pages and the urls for those pages will have this as the base. This value also gets translated into the page for the content page, which lives in `content/pages`.
+
+### Is this a form app?
+
+If this is a form, there are some more questions after this step. If not, you're done!
+
+### What's your form number?
+
+This is the form number for the paper form you're converting. Normally it's something like 22-1990 or 21-22. This is used in a couple places in the UI and also as a key for the save in progress functionality of our forms.
```
Comment (melaniegershman) [inline docs/GeneratorOptions.md]: One suggestion here would be to clarify where you might find the form number on the paper form, or even include a screenshot, like so: 
screen shot 2018-04-05 at 4 40 27 pm

File: docs/GeneratorOptions.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### What's the Google Analytics event prefix you want to use?
+
+Our shared form code sends events to Google Analytics and we need a unique prefix to be able to categorize the events by form. It's normally a dash separated value like `hca-` or `edu-1990-` and is made up by developers.
+
+### What's the respondent burden of this form in minutes?
+
+Each paper form has OMB information that lives in the margins of each page (usually in the top right). This is the respondent burden value from that information. All the information from this question and the next two can be found in the form's introduction page, passed as props to the OMBInfo component, if you need to change it later.
```
Comment (melaniegershman) [inline docs/GeneratorOptions.md]: Is it more clear to provide the file path for the Intro page, like so?:

> passed as props to the OMBInfo component **`found in containers/IntroductionPage.jsx`**

**Review by melaniegershman — APPROVED** [review]
These are really helpful updates. I made a few suggestions for your consideration below. Also, will there eventually be a troubleshooting section for issues such as the one I had earlier today, with the extra `/` in `manifest.json`'s `rootUrl`?

File: docs/GeneratorOptions.md (hunk 1 above)
Comment (melaniegershman) [reply docs/GeneratorOptions.md]: I see now that this has been handled further below!

**Comment by jbalboni** [comment]
@melaniegershman Thanks, I've made some updates. A troubleshooting section probably makes sense, I'll have to think about what goes in there. For the url issue, we have validation in the generator that should prevent it, plus I added a check for it to another PR, so that an error shows up in the console: https://github.com/department-of-veterans-affairs/vets-website/pull/7533/files#diff-188da8af3e1e05db38e141cae9f62f25R33

**Comment by melaniegershman** [comment]
Thanks @jbalboni that check & error will be super helpful!

**Pull request merged**